user-guide/Basic_Functionality/Trending/Advanced_analytics_trending/Working_with_pattern_matching.md

```diff
@@ -26,3 +28,15 @@ If you are viewing a trend graph for a single parameter, and it contains pattern
 
 > [!TIP]
 > See also: [Monitoring of trend patterns](xref:Monitoring_of_trend_patterns)
+
+## Multivariate patterns
+
+From DataMiner 10.3.8/10.4.0 onwards, DataMiner Analytics can automatically recognize recurring patterns in trend graphs showing trend information of multiple parameters. <!-- RN 36454 + 36731 + 36327 -->
+
+When you open a trend graph showing trend information for multiple parameters, you have the option to define a *multivariate trend pattern*.
+
+As with single-parameter patterns (i.e. [univariate patterns](#univariate-patterns)), you can [specify a number of options](xref:Managing_pattern_definitions) in the *Pattern Overview* window. <!-- RN 35010 -->
+
+If you are viewing a trend graph for multiple parameters in which multivariate patterns have been detected, these will be highlighted in orange when you hover the mouse pointer over the button representing a pattern, or if the option *Expand tags* is selected in the right-click menu.
```

Comment:
actually you don't necessarily need multiple parameters to be displayed in your trend graph in order to see the occurrences of a multivariate pattern. You need all parameters involved in your multivariate pattern to be loaded in the trend graph to be able to "create" the multivariate pattern. But to just see the detected pattern occurrences at a later time,  one does not necessarily need all the parameters in the graph: one can for instance open the trend graph of only one of the parameters and still see the pattern tags. As explained in your next sentence, by clicking the icon the rest of the parameters can then be loaded if one is interested in seeing which other parameters are involved in the pattern